backend/app/controllers/spree/admin/promotions_controller.rb

```diff
@@ -41,8 +41,9 @@ def collection
 
         @collection = super
         @search = @collection.ransack(params[:q])
-        @collection = @search.result(distinct: true).
+        @collection = @search.result.
           includes(promotion_includes).
+          references(promotion_references).
```

Comment:
Two thoughts:

1) Instead of `promotion_includes` and `promotion_references`, maybe we should just have `promotions_scope`
2) Maybe we should provide a config instead of requiring a monkey-patch

E.g.:
```ruby
merge(Spree::Backend::Config.promotions_index_scope).
```
?